tests/poster_mock.py

```diff
@@ -40,7 +47,40 @@ def do_GET(self):
                 self.wfile.write(content.encode("utf-8"))
                 return
 
+        def do_POST(self):
+            if re.search(self.TOKEN_PATTERN, self.path):
+                self.send_response(requests.codes.ok)
+                self.send_header("Content-Type", "application/json; charset=utf-8")
+                self.end_headers()
+                content = json.dumps(
+                    {
+                        'access_token':'861052:02391570ff9af128e93c5a771055ba88',
+                        'account_number':'861052',
```

Comment:
@paulodamaso Does it pass linter?
As I know we check double quotes before merging